src/Diagnostics/FxCop/System.Runtime.Analyzers/Test/Design/DefineAccessorsForAttributeArgumentsTests.Fixer.cs

```diff
@@ -215,6 +215,7 @@ public PublicSetterTestAttribute(string name)
     public string Name
     {
         get { return m_name; }
+
```

Comment:
The default formatting rules for C# puts a blank line between accessors. When we change the accessor, it ends up formatting that line.